CMakeLists.txt

```diff
@@ -641,7 +643,7 @@ CHECK_LIBRARY_EXISTS(${NETCDF_C_LIBRARY} oc_open "" BUILD_DAP)
 ###
 # Check to see if szip write capability is present in netcdf-c.
 ###
-SET(CMAKE_REQUIRED_INCLUDES ${NETCDF_INCLUDE_DIR})
+SET(CMAKE_REQUIRED_INCLUDES ${NETCDF_C_INCLUDE_DIR})
```

Comment:
This looks important - can you explain this change?

Reply:
Actually, the CMakeLists.txt changes are important.  Please see my first message on getting a clean build on Windows.

Reply:
This is another example of a good change that could be submitted as a separate PR.